cmd/apps/grafana_app.go

```diff
@@ -36,13 +37,28 @@ func MakeInstallGrafana() *cobra.Command {
 		const chartVersion = "5.0.4"
```

Comment:
@Waterdrips Since we are using helm version `v3.1.2` in [arkade](https://github.com/alexellis/arkade/blob/135b526943613990ba9b205e4e11dbd201ac33f5/pkg/helm/helm.go#L19). Grafana helm [charts](https://github.com/grafana/helm-charts/blob/main/charts/grafana/README.md#upgrading-an-existing-release-to-a-new-major-version) above 6.0.0 recommend helm version ` >= 3.4.0`.

Possibly @aidun can put some more lights on it.

Reply:
There are some charts that won't install with a newer helm version. The Grafana chart may recommend 3.4.0, but is it a hard requirement?

Reply:
I tried it and nothing broke in installation. Not sure, why the recommendation. helm 3.4.0 is recommended for chart from 6.0.0